### ISSUE TITLE
Widgets page: the controls in the toolbar should be in an ARIA toolbar

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#24938]
**Describe the bug**

I'm not sure why the first UI controls in the Widgets page top toolbar are within a role=menu with orientation=vertical:

<img width="879" alt="Screenshot 2020-08-31 at 16 08 09" src="https://user-images.githubusercontent.com/1682452/91731163-e6f2ed00-eba6-11ea-872d-570087f250a7.png">

The orientation isn't vertical but, more importantly, this is not a menu.

For semantics and consistency with the top toolbar in the block editor, these buttons should be in an ARIA toolbar. No more, no less than what is currently implemented in the block editor page.

**Comment by mapk** [comment]
Yes, agreed. 

cc @draganescu @talldan @adamziel please make sure the Widget editor reflects the same details (ie. ARIA) that the post editor contains. That should resolve this issue.

**Comment by afercia** [comment]
Just to note that arrow keys navigation and "roving tabindex" should be implemented as well. I guess they should come in by default with the toolbar component.
